library/include/rocrand_common.h

```diff
@@ -79,6 +65,8 @@ unsigned long long mad_u64_u32(const unsigned int x, const unsigned int y, const
     #endif
 }
 
+#undef OLDER_THAN_NAVI21
```

Comment:
This comment is not accurate and if we are dropping assembly completely, do we need it?

Reply:
I think the #undef was missed on the last PR update.

Reply:
I have to remove this.

Reply:
Removed this line.